.hound.yml

```diff
@@ -1,4 +1,6 @@
 # See https://houndci.com/configuration for help.
+fail_on_violations: true
```

Comment:
Thanks @teoljungberg My take - there is no point in having validators if you can just simply ignore them(even for the rules I don't like). It would be like having CI in place and just ignoring failing specs.